src/catalua_hooks.cpp

```diff
@@ -17,6 +17,7 @@ constexpr auto hook_names = std::array
     "on_creature_performed_technique",
     "on_creature_melee_attacked",
     "on_mapgen_postprocess",
+    "pre_death"
```

Comment:
```suggestion
    "on_before_char_death"
```
should be renamed to be more consistent with other hook names